zebra-chain/src/value_balance/tests.rs

```diff
@@ -0,0 +1,65 @@
+mod test {
```

Comment:
Can we use `tests::prop` like the rest of `zebra_chain`?

Reply:
For example, here we have `prop` in a separate file:
https://github.com/ZcashFoundation/zebra/tree/main/zebra-chain/src/work/tests

Reply:
https://github.com/ZcashFoundation/zebra/pull/2505/commits/8e4d76de53925739129b9a23c26edfaeb55e7d83